### PR TITLE
Fix 'set on destroyed object' errors

### PR DESCRIPTION
**Pull request by Gaurav0**
Also updates to Ember 2.18

### CODE REVIEW

File: addon/components/infinite-scroll-container.js [hunk 1]
```diff
@@ -54,11 +48,15 @@ export default Component.extend(InfiniteScrollMixin, {
...
+      content = newRecords;
+    }
     let infiniteContentPropertyName = this.get('infiniteContentPropertyName');
     let model = this.get(infiniteContentPropertyName);
 
     if (model) {
-      model.addObjects(newRecords.get('content'));
+      model.addObjects(content);
     } else {
       this.set(infiniteContentPropertyName, newRecords);
```
Comment (JeffLuong) [inline addon/components/infinite-scroll-container.js]: Should we `safeSet` this too?

**Comment by joebartels** [comment]
consider bumping version in package.json

File: addon/components/infinite-scroll-container.js (hunk 1 above)
Comment (Gaurav0) [reply addon/components/infinite-scroll-container.js]: I think it is unlikely the model would be destroyed, and we don't have the code yet for safeAddObjects etc.

Comment (JeffLuong) [reply addon/components/infinite-scroll-container.js]: @Gaurav0 sorry - was meant for line `63` where we're setting `infiniteContentPropertyName`

Comment (Gaurav0) [reply addon/components/infinite-scroll-container.js]: Oh, yeah, probably.

Comment (Gaurav0) [reply addon/components/infinite-scroll-container.js]: We don't actually use infiniteScrollContainer in our app, that's probably why the error didn't pop up.

**Pull request merged**